src/zappy_gui_src/Core/SFML/Interface/Inventory/Inventory.hpp

```diff
@@ -5,27 +5,65 @@
 ** Inventory
 */
 
+/// \file src/zappy_gui_src/Core/SFML/Interface/Inventory/Inventory.hpp
+
 #ifndef INVENTORY_HPP_
 #define INVENTORY_HPP_
 
 #include <SFML/Graphics.hpp>
 #include <memory>
+#include <utility>
+#include "Entity.hpp"
+
+#define FONT_PATH "assets/arial.ttf"
 
 namespace gui {
     class Inventory {
         public:
+            /// \brief constructor of the object
             Inventory();
+            /// \brief constructor of the object
```

Comment:
What’s the difference between, «  constructor of the object » x 2

Reply:
I need to use my joker @osvegn on this one